README.md

```diff
@@ -263,6 +265,60 @@ let is_sql = client.query(query_is_sql.unwrap()).unwrap();
 * [`Delete`](https://opencrux.com/reference/transactions.html#delete) - Deletes the specific document at a given valid time
 * [`Evict`](https://opencrux.com/reference/transactions.html#evict) - Evicts a document entirely, including all historical versions (receives only the ID to evict)
 * [`Match`](https://opencrux.com/reference/transactions.html#match) - Matches the current state of an entity, if the state doesn't match the provided document, the transaction will not continue
+* To create a single `Action` use the static methods for `put`, `evict`, `delete`, `match_doc`. To add a `:valid-date` to your `Action` call builder function `with_valid_date`, like `Action::put(person).with_valid_date(timed)`.
+
+[`Actions`](https://docs.rs/transistor/1.3.11/transistor/http/enum.Actions.html) is a builder struct to help you create a `Vec<Action>` for `tx_log`. Available functions are:
+* `new` static method to instantiate struct `Actions`.
+* `append_put<T: Serialize>(action: T)` appends an `Action::Put` to `Actions` with no `valid-time`.
+* `append_put_timed<T: Serialize>(action: T, date: DateTime<FixedOffset>)` appends an `Action::Put` to `Actions` with `valid-time`.
+* `append_delete(id: crate::types::CruxId)` appends an `Action::Delete` to `Actions` with no `valid-time`.
+* `append_delete_timed(id: crate::types::CruxId, date: DateTime<FixedOffset>)` appends an `Action::Delete` to `Actions` with `valid-time`.
+* `append_evict(id: crate::types::CruxId)` appends an `Action::Evict` to `Actions`.
+* `append_match_doc<T: Serialize>(id: crate::types::CruxId, action: T)` appends an `Action::Match` to `Actions` with no `valid-time`.
+* `append_match_doc_timed<T: Serialize>(id: crate::types::CruxId, action: T, date: DateTime<FixedOffset>)` appends an `Action::Match` to `Actions` with `valid-time`.
+* `build` generates the `Vec<Action>` from `Actions`
+
+```rust
```

Comment:
I think the `Database` type definition should be in this example to make it more clear.

Reply:
???